### ISSUE TITLE
Checking the performance of classifiers in high dimensional noise setting.

### ISSUE THREAD

**Issue by sahanasrihari** [neurodata/scikit-learn#1]
To test the accuracy of multiple classifiers in the presence of varying levels of high dimensional noise using some simulated datasets.
The performance measure is the accuracy of the classifiers with respect to the number of noise dimensions added to the data.

RESULT:
Comparative plot of RF, SVM and KNN for a classification task in the midst of high dimensional (high variance) noise. 
[https://github.com/NeuroDataDesign/team-forbidden-forest/blob/master/Sahana/gauss_classifiers.ipynb](url)

**Comment by bdpedigo** [comment]
This issue is unclear about what you are actually proposing to PR into sklearn. You can be a lot more detailed about the fact that you are proposing a new tutorial, what the figures will be, what the data is

**Comment by sahanasrihari** [comment]
### Description
The sklearn's example on comparing the different classifier accuracies does not have multiple settings for testing various scenarios. 
There is no concrete example showing when some of these algorithms win and when they lose. 
One scenario to consider is - given a dataset of a relatively low order dimension, how does the accuracy of classifiers change with respect to the addition of noise dimensions.

Noise dimensions are any features added across the dimensions of the dataset which is bears no relevance to the original signal dimensions. 

**Goal** 
To check the performance of Random Forest, Support Vector Machine and K Nearest Neighbours as three different classifiers for the additions of gaussian noise across three different variance values. 

**Proposed changes in the form of PR**
I am proposing a new tutorial in the form of a jupyter notebook containing all the code from data generation to the computation of accuracies across noise dimensions.
The final figure will contain a plot of the original datasets adopted from [https://scikit-learn.org/stable/auto_examples/classification/plot_classifier_comparison.html](url)
and 9 different plots of "Accuracy Vs Number of Noise Dimensions" for the 3 different datasets and 3 different variances of gaussian noise. The plot will containing the testing accuracies across 50 trials of the experiment. 




> This issue is unclear about what you are actually proposing to PR into sklearn. You can be a lot more detailed about the fact that you are proposing a new tutorial, what the figures will be, what the data is

@bdpedigo  I have made changes to the issue, please let me know if there is need of adding more detail.
